_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils_new.cpp

```diff
@@ -2305,6 +2305,15 @@ void MfxHwH264Encode::ConfigureTask(
         task.m_maxPBFrameSize = 0;
     }
 
+    if (video.mfx.RateControlMethod == MFX_RATECONTROL_CQP)
+    {
+        task.m_repackMaxIFrameSize = video.mfx.MaxIFrameSize;
+        task.m_repackMaxPFrameSize = video.mfx.MaxPFrameSize;
+        task.m_repackMaxBFrameSize = video.mfx.MaxBFrameSize;
+        task.m_repackNumPasses = video.mfx.NumPasses;
+        memcpy_s(task.m_repackDeltaQP, task.m_repackNumPasses, video.mfx.DeltaQP, task.m_repackNumPasses);
```

Comment:
Please, do not use _s functions. We also cleaned MSDK code from memcpy calls, switching to std::copy
Here you will have:
```
std::copy(std::begin(video.mfx.DeltaQP), std::begin(video.mfx.DeltaQP) +task.m_repackNumPasses, std::begin(task.m_repackDeltaQP));
```

Reply:
Will replace the memcpy_s with std::copy.